tests/telemetry/conftest.py

```diff
@@ -171,6 +171,11 @@ def test_eventd_healthy(duthosts, enum_rand_one_per_hwsku_hostname, ptfhost, set
 
     duthost = duthosts[enum_rand_one_per_hwsku_hostname]
 
+    status = duthost.get_feature_status()
+    platform = duthost.facts["platform"]
+    if platform in ['x86_64-arista_7060_cx32s'] and ('eventd' not in status or status['eventd'] == 'disabled'):
```

Comment:
If eventd is the necessary feature for this test case, is it OK to just check eventd feature status here?

Reply:
Fixed